spec/visual/app.rb

```diff
@@ -55,7 +55,13 @@ def query_string
     @title = "#{@lexer.class.tag} | Visual Test"
     @highlighted = Rouge.highlight(@sample, @lexer, @formatter)
 
-    erb :lexer
+    if params[:juxtaposed]
+      @raw = Rouge.highlight(@sample, 'plaintext', @formatter)
+      @comment_color = @theme.class.get_style(Rouge::Token::Tokens::Comment).fg
+      erb :juxtaposed, layout: nil
```

Comment:
This might be a dumb question, but why do you need to set `@raw` and `@comment_color` for the juxtaposed template if you don't for the regular lexer template? It looks visually like the main difference is how the containers sit relative to each other.

Reply:
`@raw` could be set for the default layout as well. It is not necessary though.
What is different is the value of the variable. The default layout uses `CGI.escape_html(@sample)`, this uses `Rouge.highlight(@sample, 'plaintext', @formatter)`

Setting `@comment_color` is a hack to emulate the functionality of a Sass variable :stuck_out_tongue: